frontend/src/views/chart/group/Group.vue

```diff
@@ -290,17 +290,29 @@
       class="dialog-css"
       :destroy-on-close="true"
     >
-      <el-row style="width: 400px;">
+      <el-row style="width: 800px;">
         <el-form ref="form" :model="table" label-width="80px" size="mini" class="form-item">
-          <el-form-item :label="$t('chart.view_name')">
-            <el-input v-model="chartName" size="mini" />
-          </el-form-item>
+          <el-col :span="12">
+            <el-form-item :label="$t('chart.view_name')">
+              <el-input v-model="chartName" style="height: 34px" size="mini" />
+            </el-form-item>
+          </el-col>
+          <el-col :span="12" v-if="optFrom==='panel'">
```

Comment:
Codacy has a fix for the issue: [Attribute "v-if" should go before ":span".](https://app.codacy.com/gh/dataease/dataease/pullRequest?prid=7710993)
```suggestion
          <el-col v-if="optFrom==='panel'" :span="12">
```